### ISSUE TITLE
Web release not working

### ISSUE THREAD

**Issue by UX3D-haertl** [KhronosGroup/KTX-Software#355]
When I download libktx.js and libktx.wasm from the release page, the initialization is caught up in an endless loop.
This is the loop which does not resolve in line 8078 of the prettified file:
```
Module["then"] = function(func) {
            if (calledRun) {
                func(Module)
            } else {
                var old = Module["onRuntimeInitialized"];
                Module["onRuntimeInitialized"] = function() {
                    if (old)
                        old();
                    func(Module)
                }
            }
            return Module
        }
        ;
```

The files work as intended if I build them from source myself.

**Comment by MarkCallow** [comment]
Interesting! When you build it yourself are you using the included CMake files? What version of Emscripten are you using?

**Comment by UX3D-haertl** [comment]
I just installed the latest emscripten version as described [here](https://emscripten.org/docs/getting_started/downloads.html) and build libktx via the instructions in BUILDING.md

**Comment by UX3D-haertl** [comment]
This issue persists with beta6

**Comment by MarkCallow** [comment]
Yes. Beta 6 did not change anything regarding how the web release is built. I'm looking into it now.

**Comment by MarkCallow** [comment]
Please try just released [Release Candidate 1](https://github.com/KhronosGroup/KTX-Software/releases/tag/v4.0.0-rc1). The web modules in this release have been built with the latest emsdk (2.0.16).

**Comment by UX3D-haertl** [comment]
I tried to include the new version into the gltf-sample-viewer, but I get this error:
```
Uncaught (in promise) TypeError: Cannot read property 'getContextSafariWebGL2Fixed' of null
    at Object.createContext (libktx.js:9)
    at KtxDecoder.init (ktx.js:31)
```
(I don't have safari installed)
But I get this error as well with my own build, so this is probably another issue.

**Comment by MarkCallow** [comment]
Thank you for trying. This sounds like a bug in Emscripten's OpenGL ES on WebGL emulator. I did not get this error with Emscripten 2.0.15 which is the version available when I worked on updating the Docker image and thus used in the Docker images I was building myself.

I'll try to investigate further as soon as I have time. If you have time, please investigate yourself. With KTX-Software release imminent my time is limited.

**Comment by MarkCallow** [comment]
This is indeed a bug in Emscripten. See Emscripten [issue #13296](https://github.com/emscripten-core/emscripten/issues/13926). I can't see any workaround so we'll have to wait for a fix.

**Comment by MarkCallow** [comment]
I have found a way to use the latest Docker image but build using Emscripten 2.0.15 to avoid the bug. The zip-file packaged modules produced by the modified build script pass all the tests. The same script is used to build the releases deployed on GitHub so this problem should be fixed in the next release.

**Comment by MarkCallow** [comment]
I believe this is fixed in the released v4.0.0. If not, please reopen.
